### ISSUE TITLE
exp results

### ISSUE THREAD

**Issue by ARIchMan6** [harkash/contrastive-predictive-coding-for-har#4]
I trained using the UCI dataset according to the configuration in the paper, and the results are as follows, which differ greatly from the results mentioned in the paper. Could you please explain this?

The epoch: 149 | phase: train | loss: 2.0122 | accuracy: 0.5475 | mean f1-score: 0.0000 | weighted f1-score: 0.0000
The epoch: 149 | phase: val | loss: 1.8653 | accuracy: 0.6001 | mean f1-score: 0.0000 | weighted f1-score: 0.0000
The epoch: 149 | phase: test | loss: 2.9983 | accuracy: 0.2864 | mean f1-score: 0.0000 | weighted f1-score: 0.0000
Epoch 149 completed in 0m 42s
The best epoch is 147

Currently, overfitting has occurred for many epochs. The test set performance reached 0.28 at the 98th epoch and has remained unchanged since then.
